src/Connection/HiveDsnFactory.php

```diff
@@ -27,6 +27,9 @@ public function create(
         $parameters['Port'] = $dbConfig->getPort();
         $parameters['Schema'] = $dbConfig->getDatabase();
         $parameters['UseNativeQuery'] = '1';
+        $parameters['DefaultStringColumnLength'] = '65536';
```

Comment:
Pridal som nejake uzitocne parametre do spojenia, ... nastavil som, aby boli povolene aj dlhsie stringy + aby reportovalo string typy z DB ako unicode.

Klient neodpisuje, neviem ci to pomoze danemu problemu, ale v kazdom pripade je to dobre tu mat.

Uz tam nemame velmi co ine zmenit. Ten problem (vid ZD) je blby v tom, ze to spadne iba na par riadkoch, takze mozno je to nejaky bug v databaze alebo v ODBC drivery.